www/js/main.js

```diff
@@ -116,11 +116,11 @@ angular.module('emission.main', ['emission.main.recent',
   });
 
   .state('root.main.refuse', {
-    url: '/heatmap',
+    url: '/refuse',
     views: {
-      'main-heatmap': {
-        templateUrl: 'templates/main-heatmap.html',
-        controller: 'HeatmapCtrl'
+      'refuse': {
+        templateUrl: 'templates/intro/refuse.html',
+        controller: 'refuseCtrl'
```

Comment:
I am not sure this will actually work in the absence of `refuseCtrl`.
Let me merge, try it out and then fix.